composer.json

```
@@ -18,7 +18,7 @@
         "ext-json": "*",
         "ext-tokenizer": "*",
         "composer/semver": "^3.2",
-        "composer/xdebug-handler": "^3.0",
+        "composer/xdebug-handler": "^3.0.2",
```

_minor_

```suggestion
        "composer/xdebug-handler": "^3.0.3",
```

if we need to touch it, let's use the newest ;)